jsk_2016_01_baxter_apc/node_scripts/transform_publisher.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
```

Comment:
Why not place this node to `dynamic_tf_publisher` or `jsk_topic_tools` or `tf` ROS package?